src/test/descriptor_tests.cpp

```diff
@@ -13,13 +13,15 @@
 
 namespace {
 
-void CheckUnparsable(const std::string& prv, const std::string& pub)
+void CheckUnparsable(const std::string& prv, const std::string& pub, std::string expected_error)
```

Comment:
Nit: `expected_error` could be const ref?

Reply:
Done